src/ert/analysis/_es_update.py

```diff
@@ -392,6 +399,24 @@ def _copy_unupdated_parameters(
             target_ensemble.save_parameters(parameter_group, realization, ds)
 
 
+def _save_cross_correlations_to_disk(
```

Comment:
Why is the load function is located on `local_ensemble` and the save function here?